### PR TITLE
Update the documentation to reflect the use of the admin role.

### LINKED ISSUE #3882: Update alaveteli.org documentation to reflect new roles rather than `admin_level` attribute

### PR DESCRIPTION
**Pull request by crowbot**
Fixes #3882

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -64,29 +64,60 @@ Definitions
...
 
   <dt>
-    <a name="admin">admin interface</a> (also: admin)
+    <a name="admin_interface">admin interface</a>
   </dt>
   <dd>
     The <strong>admin interface</strong> allows users who have
-    <a href="{{ page.baseurl }}/docs/glossary/#super" class="glossary__link">super</a>
-    administrator privilege to manage some aspects of how your
+    <a href="{{ page.baseurl }}/docs/glossary/#admin" class="glossary__link">admin</a>
```
Comment (stevenday) [inline docs/glossary.md]: I think this needs a "the" before the linked word "admin"

File: docs/installing/next_steps.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ title: Next Steps
...
 
 Alaveteli ships with an
 <a href="{{ page.baseurl }}/docs/glossary/#emergency" class="glossary__link">emergency user</a>
 that has access to the admin. So when you've just created a new site, you
-should sign up to create your own account, then log into admin as the emergency
+should sign up to create your own account, then log into the admin interface as the emergency
 user to promote your new account to be an administrator with
-<a href="{{ page.baseurl }}/docs/glossary/#super" class="glossary__link">super</a>
-privilege.
+<a href="{{ page.baseurl }}/docs/glossary/#admin" class="glossary__link">admin</a>
```
Comment (stevenday) [inline docs/installing/next_steps.md]: Again - should this be "with *the* admin role"?

File: docs/installing/next_steps.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ First, in the browser:
 * You're now on the Alaveteli admin page.
 * Click on **Users**  (in the navigation menu across the top of the page), and
   click on your name in the list of users. On *that* page,  click **Edit**.
-* Change your *Admin level* to "super" and click **Save**.
+* n the "Roles" section, check the "admin" checkbox and click **Save**.
```
Comment (stevenday) [inline docs/installing/next_steps.md]: missing an "i" here for "in"

File: docs/running/redaction.md [hunk 4]
```diff
@@ -216,8 +216,8 @@ spelling.
    requests on your site.
 </div>
 
-* click on **Requests** in the admin and search or find the one you want
-* click on the request's title to go to its page in the admin
+* click on **Requests** in the admin interfave and search or find the one you want
```
Comment (stevenday) [inline docs/running/redaction.md]: typo: interfave

File: docs/customising/config.md [hunk 5]
```diff
@@ -549,8 +549,8 @@ THEME_URLS:
       </p>
       <p>
         For details of this process, see
-        <a href="{{ page.baseurl }}/docs/installing/next_steps/#create-a-superuser-admin-account">creating
-          a superuser account</a>.
+        <a href="{{ page.baseurl }}/docs/installing/next_steps/#create-an-admin-account">creating
+          a admin account</a>.
```
Comment (lizconlan) [inline docs/customising/config.md]: pedant: "an admin"

**Pull request merged**